### PR TITLE
Fix pipeline and using litmus prefix in the cluster name

### PR DESCRIPTION
**Pull request by uditgaurav**
- Fix pipeline and using litmus prefix in the cluster name
Signed-off-by: Udit Gaurav <uditgaurav@gmail.com>

### CODE REVIEW

File: tests/openebs-pool-container-failure_test.go [hunk 1]
```diff
@@ -164,7 +164,6 @@ var _ = Describe("BDD of openebs pool container failure experiment", func() {
 			//Fetching the runner pod and Checking if it get in Running state or not
 			By("Updating the result table")
 			By("Wait for chaso-runner to come in running sate")
```
Comment (ispeakc0de) [inline tests/openebs-pool-container-failure_test.go:166]: Wait for chaos-runner to come in running state"

File: tests/openebs-pool-container-failure_test.go [hunk 2]
```diff
@@ -164,7 +164,6 @@ var _ = Describe("BDD of openebs pool container failure experiment", func() {
...
-			pipelineResult, err := utils.ResultUpdate(experimentName, engineName, clientSet)
```
Comment (ispeakc0de) [inline tests/openebs-pool-container-failure_test.go:167]: This is not required?

Comment (uditgaurav) [reply tests/openebs-pool-container-failure_test.go:167]: typo

Comment (ispeakc0de) [reply tests/openebs-pool-container-failure_test.go:167]: Why are we not updating result table for this experiment?

File: tests/openebs-pool-container-failure_test.go [hunk 3]
```diff
@@ -162,9 +162,7 @@ var _ = Describe("BDD of openebs pool container failure experiment", func() {
 			time.Sleep(2 * time.Second)
 
 			//Fetching the runner pod and Checking if it get in Running state or not
-			By("Updating the result table")
-			By("Wait for chaso-runner to come in running sate")
-			pipelineResult, err := utils.ResultUpdate(experimentName, engineName, clientSet)
+			By("Wait for chaos-runner to come in running sate")
```
Comment (ispeakc0de) [inline tests/openebs-pool-container-failure_test.go:165]: state?

File: tests/openebs-pool-container-failure_test.go (hunk 2 above)
Comment (uditgaurav) [reply tests/openebs-pool-container-failure_test.go:167]: @ispeakc0de  pls see line 249

**Pull request merged**